src/gpodder/util.py

```diff
@@ -595,6 +595,19 @@ def delete_file(filename):
         pass
 
 
+def is_html(text):
```

Comment:
Sorry: I missed your PR.
I started from the regexp you pointed to in a previous comment (`r'<[a-z][\s\S]*>', re.IGNORECASE`), but found `[\s\S]*` equivalent to `.*` (isn't it?) and found no reason to not restrict the element name, only accepting unspecified characters after a space (`(\\s.*)?`). This way, `a<b<c` is rejected, for instance.
It can use the ignore case flag: `'<[a-z][a-z0-9]*(\\s.*)?>', re.IGNORECASE`. It's a gross over approximation anyway, so I'm OK with using `r'<[a-z].*>', re.IGNORECASE`.